presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraTable.java

```diff
@@ -49,6 +51,31 @@ public CassandraTableHandle getTableHandle()
                 .collect(toList());
     }
 
+    public List<CassandraColumnHandle> getClusteringKeyColumns()
+    {
+        return columns.stream()
+                .filter(CassandraColumnHandle::isClusteringKey)
+                .collect(toList());
+    }
+
+    public boolean isLastClusteringKeyColumn(CassandraColumnHandle columnHandle)
```

Comment:
Do we really need this method? Can't we assume that clustering columns returned by `getClusteringKeyColumns` is in correct order? And check it in the loop that it is last element?